mainnet-1/specs/cosmoshub.json

```diff
@@ -0,0 +1,159 @@
+{
```

Comment:
missing cosmos hub testnet spec, please add it too

Reply:
In mainnet we would have mainnet and testnet providers? I thought would be best to keep testnet in testnet and mainnet in mainnet

Reply:
Do we plan to use archive extension for testnet spec?

Reply:
@RuslanGlaznyov we will have both and monetize both with pools according to the rate the ecosystem wants, testnets need paid rpc too :)